### ISSUE TITLE
Template thumbnails are cut off in Add From Template modal

### ISSUE THREAD

**Issue by PatrickA-RiseVision** [Rise-Vision/rise-vision-apps#2466]
### What is the expected behavior?
Add From Template modal's template thumbnails keep their proportions so they are visible in the gallery.

### What is the actual behavior?
The Add From Template modal now scales to fullscreen, but the thumbnails do not keep their proportions causing them to be cut off at the top and bottom.

### What steps can be taken to replicate the problem?
1. Go to Presentations.
2. Click Add From Template.

If you have a normal 16:9 monitor:
![image](https://user-images.githubusercontent.com/40774237/108279052-5dce7300-7152-11eb-96c9-0554b6dc45f3.png)

If you're Kaden and have some crazy aspect ratio (on arch btw):
![image](https://user-images.githubusercontent.com/40774237/108279131-7b9bd800-7152-11eb-9af0-3ff63663c545.png)

**Comment by mmeiers** [comment]
@osmankarame didnt you have some thoughts on this?

**Comment by osmankarame** [comment]
Hi! I'm not sure I'd put auto to fill up the full size of the screen. I still think we should have a little space, so I think 90% or width should be fine. There is also another solution to keep the aspect ratio of the image with "object-fit:cover" on the img tag.

**Comment by ezequielc** [comment]
@osmankarame Sounds good, thanks for the feedback!

**Comment by mmeiers** [comment]
@ezequielc FYI We are noticing some odd issues with all modals. There is no padding on the sides anyomre. It gives me the impression there was a recent release that had some unintended effects. https://www.screencast.com/t/ZMqVUm5xCu1 Because of that I might suggest re-evaluating the date to potentially roll back whatever did this temporarily.

**Comment by ezequielc** [comment]
@mmeiers @PatrickA-RiseVision Thanks for the heads up, we did introduce an issue causing modals to expand to the entire width of the screen. We reverted the change, so now modal have again a limited width and the thumbnails won't cut off anymore.

However, we noticed that scrolling bars can appear if screen height is too small, we we will keep this defect open until our next cooldown to try to fix that too. Thanks!
